examples/using-asciidoc/src/templates/article.js

```diff
@@ -98,6 +113,9 @@ export const pageQuery = graphql`
         authorInitials
         email
       }
+      frontmatter {
+        source
```

Comment:
none of the example asciidoctor files have frontmatter yet right?

Reply:
oh shoot i forgot to git add, which explained why ci failed